README.md

```diff
@@ -1,164 +1,101 @@
+Outputs                                                                                                                                                           
+------------------------------------------------------------------------------------------------------------------------------------------------
+Key                 CartApi                                                                                                                                       
+Description         API Gateway endpoint URL for Prod stage for Cart Service                                                                                      
+Value               https://0fi59sk3kb.execute-api.localhost.localstack.cloud:4566/Prod                                                                           
+------------------------------------------------------------------------------------------------------------------------------------------------
 
-Build and deploy the resources:  
-``` bash
-make backend  # Creates S3 bucket if not existing already, then deploys CloudFormation stacks for authentication, a 
-product mock service and the shopping cart service.  
+Successfully created/updated stack - aws-serverless-shopping-cart-shoppingcart-service in us-east-1
 ```
 
-### Run the Frontend Locally
+> If the SAM application build fails, you can check if you have the Python 3.8 installed and in the `PATH`. Alternatively, use [`pyenv`](https://github.com/pyenv/pyenv) to switch between multiple versions of Python.
 
-Start the frontend locally:  
-``` bash
-make frontend-serve  # Retrieves backend config from ssm parameter store to a .env file, then starts service.  
-```
+### Setting up the front-end
 
-Once the service is running, you can access the frontend on http://localhost:8080/ and start adding items to your cart. 
-You can create an account by clicking on "Sign In" then "Create Account". Be sure to use a valid email address as 
-you'll need to retrieve the verification code.
+To set up the front-end, run the following command:
 
-**Note:** CORS headers on the backend service default to allowing http://localhost:8080/. You will see CORS errors if 
-you access the frontend using the ip (http://127.0.0.1:8080/), or using a port other than 8080.  
-
-### Clean Up
-Delete the CloudFormation stacks created by this project:
-``` bash
-make backend-delete
+```shell
+make frontend-serve 
 ```
 
-## Option 2 - Automatically deploy backend and frontend using Amplify Console
+The above command will install the dependencies, run a Node.js script to retrieve backend configuration from the SSM parameter and store to a .env.local` file. The application is then served locally at `http://localhost:8080`.
+
+> CORS headers on the backend service default to allowing `http://localhost:8080/`. You will see CORS errors if 
+you access the frontend using the IP (`http://127.0.0.1:8080/`) or a port other than `8080`.
```

Comment:
I witnessed some issues when using nodejs 18. Could we please add a troubleshooting section here? 🙏 

E.g.:
If the deployment of the frontend fails with an error message similar to:
```
opensslErrorStack: [ 'error:03000086:digital envelope routines::initialization error' ],
  library: 'digital envelope routines',
  reason: 'unsupported',
  code: 'ERR_OSSL_EVP_UNSUPPORTED'
```
Please run: `export NODE_OPTIONS=--openssl-legacy-provider` and then `make frontend-serve`